packages/plugin-ext/src/plugin/output-channel/output-channel-item.ts

```diff
@@ -65,7 +65,7 @@ export class OutputChannelImpl implements theia.OutputChannel {
         this.proxy.$close(this.name);
     }
 
-    private validate(): void {
+    protected validate(): void {
```

Comment:
This one is breaking, too, as a subclass might already have this method.

Reply:
Yes thanks, I already added it to the Changelog breaking changes with the last commit.